pkg/simple/client/openpitrix/helmwrapper/helm_wrapper.go

```diff
@@ -49,11 +54,11 @@ type releaseStatus struct {
 // Info describes release information.
 type Info struct {
 	// FirstDeployed is when the release was first deployed.
-	FirstDeployed time.Time `json:"first_deployed,omitempty"`
+	FirstDeployed helmtime.Time `json:"first_deployed,omitempty"`
```

Comment:
Please keep the mind the potential compatibility issues here.

Reply:
I have removed this code.